src/instructlab/model/chat.py

```diff
@@ -61,6 +62,24 @@
 
 PROMPT_PREFIX = ">>> "
 
+DEFAULT_ENDPOINT = cfg.get_api_base(cfg._serve().host_port)
+
+
+def is_openai_server_and_serving_model(
+    endpoint: str, api_key: str, http_params: HttpClientParams
+) -> bool:
+    """
+    Given an endpoint, returns whether or not the server is OpenAI-compatible
+    and is actively serving at least one model.
+    """
+    try:
+        models = ilabclient.list_models(
+            endpoint, api_key=api_key, http_client=http_client(http_params)
+        )
+        return len(models) > 0
+    except openai.OpenAIError:
```

Comment:
`list_models` raises `ClientException`, not `OpenAIError`.

Reply:
Good catch, thanks!